polls/views.py

```diff
@@ -74,3 +75,18 @@ def vote(request, question_id):
                 'question': question,
                 'error_message': _("You are only allowed to vote once."),
             })
+
+
+def free_text_vote(request, question_id):
+    question = get_object_or_404(FreeTextQuestion, pk=question_id)
+    obj, created = FreeTextVote.objects.get_or_create(
```

Comment:
This doesn't set the answer  and so the vote will always be blank. Also it should be covered by the tests